fcl/g4/larg4_icarus.fcl

```diff
@@ -73,8 +79,22 @@ physics:
   end_paths: [ stream1 ]
 }
 
+# Store MCParticleLite in G4 to store dropped particles from KeepEMShowerDaughters: false
+physics.producers.largeant.StoreDroppedMCParticles: false
+#services.ParticleListAction.KeepDroppedParticlesInVolumes: ["volDetEnclosure"] 
+
+# ------------------------------------------------------------------------------
+# Configure mcreco to read SEDLite instead of SED and MCParticleLite in addition to MCParticle
 physics.producers.mcreco.G4ModName: "simdrift"
+physics.producers.mcreco.SimChannelLabel: "sedlite"
 physics.producers.mcreco.MCParticleLabel: "largeant"
-physics.producers.mcreco.SimChannelLabel: "simdrift"
+physics.producers.mcreco.UseSimEnergyDepositLite: true
+physics.producers.mcreco.UseSimEnergyDeposit: false
+physics.producers.mcreco.IncludeDroppedParticles: false
+physics.producers.mcreco.MCRecoPart.SavePathPDGList: [13,-13,211,-211,111,311,310,130,321,-321,2212,2112,2224,2214,2114,1114,3122,1000010020,1000010030,1000020030,1000020040]
+
+#info reducer
+physics.producers.sedlite.SimEnergyDepositLabel: "largeant:LArG4DetectorServicevolTPCActive"
+physics.producers.sedlite.useOrigTrackID: true #needed since origTrackID not filled for sedlite in SBND
```

Comment:
Comment should read "needed to store geant4 track ids" (sorry I know this comment is from my fcls)

Reply:
Done!